src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -83,10 +83,10 @@ class EventEntries extends React.Component {
 
     analytics('issue_error_banner.viewed', {
       org_id: parseInt(orgId, 10),
-      platform: project.platform,
       group: event.groupID,
       error_type: errorTypes,
       error_message: errorMessages,
+      ...(platform && {platform: project.platform}),
```

Comment:
I think evan means `platform` is `undefined`

```suggestion
      ...(project.platform && {platform: project.platform}),
```

Reply:
ouf- thanks

Reply:
I'd be okay with null traversal operator here too ;)

`platform: project?.platform`

Reply:
Just a bit more context on the reasoning here: `reload` does a check on the type and sends a sentry warning if there is a mismatch (this particular one has a ton).